content/administration/install/deploy.rst

```diff
@@ -388,20 +388,109 @@ The solutions to support livechat/motifications in a WSGI application are:
   with ``/longpolling/`` to
   :option:`the longpolling port <odoo-bin --longpolling-port>`.
 
+.. _streaming:
```

Comment:
To avoid conflicts with future anchors, I recommend using the following (although it's not in the guidelines)

[rst filename]/[subject]

```suggestion
.. _deploy/streaming:
```